content/app/development/configuration/filescan/_index.en.md

```diff
@@ -0,0 +1,86 @@
+---
+title: FileScan
```

Comment:
Wondering if we should call it malware scan instead? Or virus scan ? Might make more sense for the app developers?